ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/TargetDefinitionManager.java

```diff
@@ -78,10 +78,10 @@ public int compare(IConfigurationElement o1, IConfigurationElement o2) {
 			}
 
 			private String getString(IConfigurationElement elem) {
-				String name = elem.getAttribute("name"); //$NON-NLS-1$
+				StringBuilder name = new StringBuilder().append(elem.getAttribute("name")); //$NON-NLS-1$
```

Comment:
A StringBuilder is not necessary for static string concatenation. The Java compiler does even smarter things under the hood any ways.
```suggestion
				String name = elem.getAttribute("name"); //$NON-NLS-1$
```